lib/xnvme_be_spdk_dev.c

```diff
@@ -299,6 +299,9 @@ _xnvme_be_spdk_ident_to_trid(const struct xnvme_ident *ident, struct xnvme_opts
 		if (sscanf(ident->uri, "%[^:]:%d", addr, &port) != 2) {
 			XNVME_DEBUG("FAILED: uri: %s, trtype: %s", ident->uri,
 				    spdk_nvme_transport_id_trtype_str(trtype));
+			// Prevent assertion 0, in spdk_nvme_trid_populate_transport()
+			// TODO: find a better way to fix this
+			trid->trtype = trtype;
```

Comment:
This looks fine to me, but I'm wondering if it makes sense to move this up, so we always set `trid->trtype`. What do you think?

Reply:
Correct, it makes more sense.